src/Ledger/Deleg.lagda

```diff
@@ -166,17 +168,17 @@ data _⊢_⇀⦇_,CERT⦈_ : CertEnv → CertState → DCert → CertState → S
   CERT-deleg :
     pp ⊢ stᵈ ⇀⦇ dCert ,DELEG⦈ stᵈ'
     ────────────────────────────────
-    ⟦ e , pp , vs ⟧ᶜ ⊢ st ⇀⦇ dCert ,CERT⦈ record st { dState = stᵈ' }
-
-  CERT-vdel :
-    Γ ⊢ stᵍ ⇀⦇ dCert ,GOVCERT⦈ stᵍ'
-    ────────────────────────────────
-    Γ ⊢ st ⇀⦇ dCert ,CERT⦈ record st { gState = stᵍ' }
+    ⟦ e , pp , vs ⟧ᶜ ⊢ ⟦ stᵈ , stᵖ , stᵍ ⟧ᶜ ⇀⦇ dCert ,CERT⦈ ⟦ stᵈ' , stᵖ , stᵍ ⟧ᶜ
```

Comment:
This was a bug (`stᵍ` was quantified universally instead of pinned to `st`). Rewrote matching on the state to make it clearer. Possibly we don't want to overload the constructor for the environment and the state...